src/handlers/get_public_key.rs

```diff
@@ -64,11 +43,11 @@ pub fn handler_get_public_key(comm: &mut Comm, display: bool) -> Result<(), AppS
 
     comm.append(&[pk.pubkey.len() as u8]);
     comm.append(&pk.pubkey);
-    // Rust SDK key derivation API does not return chaincode yet
-    // so we just append a dummy chaincode.
-    const CHAINCODE_LEN: usize = 32;
-    comm.append(&[CHAINCODE_LEN as u8]); // Dummy chaincode length
-    comm.append(&[0u8; CHAINCODE_LEN]); // Dummy chaincode
+
+    const CHAINCODE_LEN: u8 = 32;
+    let code = cc.unwrap();
```

Comment:
We can safely use an unwrap here ? No need to propagate an error in case it cannot unwrap ?

Reply:
Yes it is safe for `Secp256k1`, `Secp256r1`and `Ed25519`
See https://github.com/LedgerHQ/ledger-device-rust-sdk/blob/610a73b500730b28c0b8c1b556e089a6b102d7c6/ledger_device_sdk/src/ecc.rs#L469